### PR TITLE
[WIP] Looker integration

### PR DESCRIPTION
**Pull request by pawelpinkos**
`Looker integration`

Resolves `<issue nr here>`

---
Keep in mind:
- [ ] Documentation updates
- [ ] [Changelog](CHANGELOG.md) updates

### CODE REVIEW

File: data_pipelines_cli/cli_commands/bi.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    default=False,
+    help="Deploy to BI",
+)
+@click.option(
+    "--key-path",
+    type=str,
+    required=False,
+    help="Path to the key with write access to repo with published BI code",
+)
+def bi_command(env: str, generate_code: bool, deploy: bool, key_path: str) -> None:
```
Comment (p-pekala) [inline data_pipelines_cli/cli_commands/bi.py:61]: why we need it as separate command?

File: data_pipelines_cli/cli_commands/compile.py [hunk 2]
```diff
@@ -146,7 +151,13 @@ def compile_project(
 @click.option(
     "--docker-args", type=str, required=False, help="Args required to build project in json format"
 )
+@click.option(
+    "--bi-build",
```
Comment (p-pekala) [inline data_pipelines_cli/cli_commands/compile.py:155]: I'm thinking about turning it on and off from bi.yml instead. Maybe both could be used. We would like to control it without changes in CICD process :)

**Review by p-pekala — CHANGES_REQUESTED** [review]

File: setup.py [hunk 3]
```diff
@@ -18,6 +18,8 @@
     "fsspec",
     "packaging>=20.4,<21.0",
     "colorama==0.4.4",
+    "dbt2looker==0.10.0",
```
[the remark this answers is not in the document]
Comment (pawelpinkos) [reply setup.py]: agree, done

File: data_pipelines_cli/cli_commands/deploy.py [hunk 4]
```diff
@@ -156,12 +169,26 @@ def _sync_bucket(self) -> None:
...
     help="Whether to ingest DataHub metadata",
 )
+@click.option(
+    "--bi-push",
+    is_flag=True,
+    default=False,
+    help="Whether to push to BI",
+)
+@click.option(
+    "--key-path",
```
[the remark this answers is not in the document]
Comment (pawelpinkos) [reply data_pipelines_cli/cli_commands/deploy.py]: done

File: data_pipelines_cli/cli_commands/bi.py (hunk 1 above)
Comment (pawelpinkos) [reply data_pipelines_cli/cli_commands/bi.py]: we don't actually need it. removed. bi moved into bi_utils into main module

File: tests/cli_commands/test_deploy.py [hunk 5]
```diff
@@ -152,15 +154,15 @@ def test_no_datahub_method(self):
             "pathlib.Path.cwd", lambda: self.dbt_project_config_dir
         ):
             with self.assertRaises(DependencyNotInstalledError):
-                DeployCommand("base", False, self.storage_uri, self.provider_args, True).deploy()
+                DeployCommand("base", False, self.storage_uri, self.provider_args, True, False, None).deploy()
```
[the remark this answers is not in the document]
Comment (pawelpinkos) [reply tests/cli_commands/test_deploy.py]: bi configs moved to bi.yml, no longer needed more than one param for bi

**Review by p-pekala — APPROVED** [review]

**Review by p-pekala — APPROVED** [review]

**Pull request merged**